### PR TITLE
Rename hyperbee plugin to codemon

### PR DESCRIPTION
**Pull request by Id3aFly**
This PR follows the PR #1133 and just changes the name of the plugin.

Since the name "HyperBee" originates from the combination of this plugin and some unpublished tools (which are included in my thesis, but not part of this PR), it is necessary to rename this drakvuf plugin to better differentiate between the parts of my thesis work.

My whole thesis is named "HyperBee" and the code of PR #1133  is just a part of the original code. Therefore, I would like to rename the plugin here.

### CODE REVIEW

**Comment by tklengyel** [comment]
May I suggest the name `HyperMon` instead? Most of the "monitor" plugins are just called "mon", so that would make this one consistent with the naming scheme.

**Comment by icedevml** [comment]
I think `hyper` doesn't bring any insight here because everything is `hyper` in DRAKVUF. Since it monitors code/execution, maybe `execmon`? This might sound too similar to `exmon` though.

**Comment by icedevml** [comment]
And generally I would say don't rename until it's really necessary.

**Comment by Id3aFly** [comment]
For my thesis it is required to rename it.
What do you think of "PageExecMon"?

**Comment by icedevml** [comment]
Or maybe `codemon`? [Machine] Code Mon[itor]

**Comment by Id3aFly** [comment]
Hmm, I get your point and like the idea of machine code, but I guess _codemon_ sounds very generic when read first. What do you think of modCodeMon? _mod_[ified machine] _Code Mon_[itor]?

**Comment by tklengyel** [comment]
I think `codemon` is fine

**Comment by Id3aFly** [comment]
> I think `codemon` is fine

Ok, renaming right now.

File: src/plugins/plugins.h [hunk 1]
```diff
@@ -250,7 +250,7 @@ static const bool drakvuf_plugin_os_support[__DRAKVUF_PLUGIN_LIST_MAX][VMI_OS_WI
     [PLUGIN_PROCDUMP]     = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
     [PLUGIN_RPCMON]       = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
     [PLUGIN_TLSMON]       = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
-    [PLUGIN_HYPERBEE]     = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
+    [PLUGIN_CODEMON]     = { [VMI_OS_WINDOWS] = 1, [VMI_OS_LINUX] = 0 },
```
Comment (tklengyel) [inline src/plugins/plugins.h:253]: Looks like it's misaligned now due to the name being 1-char shorter then before.

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Pull request merged**